presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorMetadata.java

```diff
@@ -1021,4 +1050,54 @@ default MaterializedViewFreshness getMaterializedViewFreshness(ConnectorSession
     {
         return new MaterializedViewFreshness(false);
     }
+
+    /**
+     * Redirects to another table or view which may or may not in the same catalog.
+     * <p>
+     * This method is called whenever the engine references a table or view name, including both accessing and creating one. Connectors are
+     * expected to return {@link Optional#empty()} if the redirection doesn't happen. The engine calls this method in an iterative manner,
+     * i.e., using the redirected name as the argument for the next call, until an {@link Optional#empty()} is returned. If a redirection
+     * does happen, the engine will access the redirected table or view name instead of the original one and raise a REDIRECTED_TABLE warning.
+     * </p>
+     * <p>
+     * Here are some caveats worth noting:
```

Comment:
> worth noting

Obviously those _not_ worth noting are not noted here :) 

```suggestion
     * <em>Caveats</em>:
```